CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## vNext (TBD)
 
 ### Enhancements
-* None
+* The ECMAScript Array APIs (such as `map`, `every`, access using indexing operator `[]`, etc.) on the `BaseSubscriptionSet` type was deprecated.
```

Comment:
Should it be moved to a different section?

Reply:
I agree, you should create a "Deprecation" section (see v10.10.0 for an example)